orttraining/orttraining/core/framework/ortmodule_graph_builder.h

```diff
@@ -28,7 +28,11 @@ struct OrtModuleGraphBuilderConfiguration {
   bool use_invertible_layernorm_grad = false;
   bool build_gradient_graph = true;
 
+  // Graph transformer configuration
   TrainingSession::TrainingConfiguration::GraphTransformerConfiguration graph_transformer_config{};
+
+  // Log severity
+  logging::Severity loglevel{logging::Severity::kWARNING};
```

Comment:
see session_option.h, we can use int type here, and skip the trouble for pybind. 

`  int session_log_severity_level = -1;`

Reply:
please see the reply below.